ftw/simplelayout/aliasblock/contents/aliasblock.py

```diff
@@ -17,8 +17,15 @@ class AliasBlockSelectable(DefaultSelectable):
 
     def is_selectable(self):
         selectable = super(AliasBlockSelectable, self).is_selectable()
-        return selectable
+        is_sl_page = self.content.portal_type == 'ftw.simplelayout.ContentPage'
 
+        # Don't allow sl pages containing another AliasBlock
+        if is_sl_page:
+            return not bool(filter(
+                lambda item: item.portal_type == 'ftw.simplelayout.AliasBlock',
```

Comment:
Here again, why not use the interface instead of the name, in case some integration needs an object inheriting from the alias block.

Reply:
There will be no other AliasBlock. If you wanna change the behavior of the AliasBlock, you can change the `Scheme` and `Class` in the FTI, but it should not change the name.